mm2src/mm2_main/src/wasm_tests.rs

```diff
@@ -160,7 +163,22 @@ async fn trade_base_rel_electrum(
 #[wasm_bindgen_test]
 async fn trade_test_rick_and_morty() {
     let bob_policy = Mm2InitPrivKeyPolicy::Iguana;
-    let alice_policy = Mm2InitPrivKeyPolicy::GlobalHDAccount(0);
+    let alice_policy = Mm2InitPrivKeyPolicy::GlobalHDAccount;
+    let alice_path_to_address = StandardHDCoinAddress {
```

Comment:
StandardHDCoinAddress::default()

Reply:
I actually did this deliberately to show which address we use and to be easily changeable.